### PR TITLE
Fix automation add TCA dialog sometimes not opening

### LINKED ISSUE #29747: Automation and Script editors are not functioning correctly
### Checklist

- [x] I have updated to the latest available Home Assistant version.
- [x] I have cleared the cache of my browser.
- [x] I have tried a different browser to see if it is related to my browser.
- [ ] I have tried reproducing the issue in [safe mode](https://www.home-assistant.io/blog/2023/11/01/release-202311/#restarting-into-safe-mode) to rule out problems with unsupported custom resources.

### Describe the issue you are experiencing

The editors have a few issues that only seem appear in firefox, trying in edge does not show the same issues. 

1.  When adding an action or condition the ui jumps to the top of the page then displays the modal, and only jumps back down to the point you're editing after you close the side menu. making it difficult to follow the flow of the logic. Clearing the cache does not fix this.
2. Sometimes, after adding an action or condition, those buttons stop working and won't let you add another. You have to save the page, and reload to add another then the same thing happens again. clearing the cache fixes this temporarily but It does come back without any updates to home assistant or the browser. When this happens the same js error message is spammed to the console over and over again without any user input(see below)
3. when problem 2 happens the back button at the top becomes seemingly unresponsive. you have to click it as many times as you clicked the broken buttons before it works but it then takes you back to the previous page without the search results if you originally searched for something.

### Describe the behavior you expected

1. it should stay at the point where the user is in the page as it does on edge.
2. the buttons should always work as it does on edge.
3. it should work as it does on edge.

### Steps to reproduce the issue

1. Go to the automation or script editor
---
### For the modal

1. scroll down the page
2. click on add action or condition
3. page scrolls back up to to and displays modal
4. stays at the top when you make your selections and it opens the side bar
5. scrolls back down when you close the side bar

---
### For the buttons not working

1. click on any add action or condition button
2. make your selections
3. close the side bar
4. click on any add action or condition button
5. nothing happens

---
### For the back button

1.  do the above then click on the back button
2. nothing happens

### What version of Home Assistant Core has the issue?

2026.2.3

### What was the last working version of Home Assistant Core?

_No response_

### In which browser are you experiencing the issue?

Firefox 147.0.4

### Which operating system are you using to run this browser?

Windows 11

### State of relevant entities

```txt

```

### Problem-relevant frontend configuration

```yaml

```

### JavaScript errors shown in your browser console/inspector

```txt
Uncaught (in promise) TypeError: can't access property "members", se[e].collections[i].groups[t] is undefined
    _getGroups add-automation-element-dialog.ts:887
    _getGroupItems add-automation-element-dialog.ts:1092
    a memoize-one.esm.js:37
    _getItems add-automation-element-dialog.ts:864
    _renderContent add-automation-element-dialog.ts:709
    render add-automation-element-dialog.ts:482
    update lit-element.ts:166
    performUpdate reactive-element.ts:1514
    scheduleUpdate reactive-element.ts:1410
    _$EP reactive-element.ts:1382
    requestUpdate reactive-element.ts:1321
    set reactive-element.ts:789
    _fetchManifests add-automation-element-dialog.ts:409
    showDialog add-automation-element-dialog.ts:283
    e card-mod.js:1
    d card-mod.js:1
    P/$/< card-mod.js:1
    E card-mod.js:1
    j card-mod.js:1
    a card-mod.js:1
    M card-mod.js:1
    M card-mod.js:1
    value card-mod.js:1
    t card-mod.js:1
    d make-dialog-manager.ts:142
    b make-dialog-manager.ts:227
    a fire_event.ts:75
    n show-add-automation-element-dialog.ts:17
    _addActionDialog ha-automation-action.ts:210
    handleEvent lit-html.ts:2114
add-automation-element-dialog.ts:887:18
    performUpdate https://homeassistant.local:8123/frontend_latest/app.6b3133e6c6d8ae29.js:78
    scheduleUpdate https://homeassistant.local:8123/frontend_latest/app.6b3133e6c6d8ae29.js:78
    _$EP https://homeassistant.local:8123/frontend_latest/app.6b3133e6c6d8ae29.js:78
    AsyncFunctionThrow self-hosted:784
    (Async: async)
    requestUpdate https://homeassistant.local:8123/frontend_latest/app.6b3133e6c6d8ae29.js:78
    set https://homeassistant.local:8123/frontend_latest/app.6b3133e6c6d8ae29.js:78
    _fetchManifests https://homeassistant.local:8123/frontend_latest/61864.10021898243e8a94.js:175
    AsyncFunctionNext self-hosted:780
    (Async: async)
    showDialog https://homeassistant.local:8123/frontend_latest/61864.10021898243e8a94.js:175
    e https://homeassistant.local:8123/hacsfiles/lovelace-card-mod/card-mod.js?hacstag=190927524421:1
    d https://homeassistant.local:8123/hacsfiles/lovelace-card-mod/card-mod.js?hacstag=190927524421:1
    P/$/< https://homeassistant.local:8123/hacsfiles/lovelace-card-mod/card-mod.js?hacstag=190927524421:1
    E https://homeassistant.local:8123/hacsfiles/lovelace-card-mod/card-mod.js?hacstag=190927524421:1
    j https://homeassistant.local:8123/hacsfiles/lovelace-card-mod/card-mod.js?hacstag=190927524421:1
    a https://homeassistant.local:8123/hacsfiles/lovelace-card-mod/card-mod.js?hacstag=190927524421:1
    M https://homeassistant.local:8123/hacsfiles/lovelace-card-mod/card-mod.js?hacstag=190927524421:1
    M https://homeassistant.local:8123/hacsfiles/lovelace-card-mod/card-mod.js?hacstag=190927524421:1
    value https://homeassistant.local:8123/hacsfiles/lovelace-card-mod/card-mod.js?hacstag=190927524421:1
    t https://homeassistant.local:8123/hacsfiles/lovelace-card-mod/card-mod.js?hacstag=190927524421:1
    d https://homeassistant.local:8123/frontend_latest/app.6b3133e6c6d8ae29.js:14
    InterpretGeneratorResume self-hosted:1312
    AsyncFunctionNext self-hosted:780
    (Async: async)
    b https://homeassistant.local:8123/frontend_latest/app.6b3133e6c6d8ae29.js:14
    a https://homeassistant.local:8123/frontend_latest/app.6b3133e6c6d8ae29.js:2
    n https://homeassistant.local:8123/frontend_latest/9640.95bece1368dac9fd.js:13
    _addActionDialog https://homeassistant.local:8123/frontend_latest/30513.49d934701b5e161c.js:10
    handleEvent https://homeassistant.local:8123/frontend_latest/app.6b3133e6c6d8ae29.js:78
```

### Additional information

_No response_

### LINKED ISSUE #30300: Automation editor becoming unresponsive on Firefox browser
### Checklist

- [x] I have updated to the latest available Home Assistant version.
- [x] I have cleared the cache of my browser.
- [x] I have tried a different browser to see if it is related to my browser.
- [ ] I have tried reproducing the issue in [safe mode](https://www.home-assistant.io/blog/2023/11/01/release-202311/#restarting-into-safe-mode) to rule out problems with unsupported custom resources.

### Describe the issue you are experiencing

The buttons for adding a trigger, a condition or an action become unresponsive in the automation editor when using Firefox browser. I suspect this problem is related to Firefox, as I use Firefox too, and, for example, everyone in this Reddit thread who’s experiencing the same issue also uses Firefox: https://www.reddit.com/r/homeassistant/comments/1rf8b5z/issues_with_automations_ui_buttons_not_doing/

The problem can be temporarily resolved by refreshing the page.

Also I tested it in Microsoft Edge and didn't have any problems.

### Describe the behavior you expected

I expected the buttons to work properly.

### Steps to reproduce the issue

1. create a new automation using Firefox browser
2. add a "choose" condition block
3. expand option 1
4. add any action to it
5. add another option and expand it
6. add any action to it
7. save the automation
8. all the "add trigger", "add condition" and "add action" buttons are unresponsive now

### What version of Home Assistant Core has the issue?

core-2026.3.3

### What was the last working version of Home Assistant Core?

_No response_

### In which browser are you experiencing the issue?

Firefox 148.0.2 (64-Bit)

### Which operating system are you using to run this browser?

Windows 11

### State of relevant entities

```txt

```

### Problem-relevant frontend configuration

```yaml

```

### JavaScript errors shown in your browser console/inspector

```txt

```

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by wendevlin**
## Proposed change
- Hopefully fix
  - fix #30300
  - fix #29747

## Type of change

- [ ] Dependency upgrade
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:
- Link to developer documentation pull request:
- Link to backend pull request:

## Checklist

- [ ] I understand the code I am submitting and can explain how it works.
- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] I have followed the [development checklist][dev-checklist]
- [ ] I have followed the [perfect PR recommendations][perfect-pr]
- [ ] Any generated code has been carefully reviewed for correctness and compliance with project standards.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

To help with the load of incoming pull requests:

- [ ] I have reviewed two other [open pull requests][prs] in this repository.

[prs]: https://github.com/home-assistant/frontend/pulls?q=is%3Aopen+is%3Apr+-author%3A%40me+-draft%3Atrue+sort%3Acreated-desc+review%3Anone+-status%3Afailure

[dev-checklist]: https://developers.home-assistant.io/docs/development_checklist/
[docs-repository]: https://github.com/home-assistant/home-assistant.io
[perfect-pr]: https://developers.home-assistant.io/docs/review-process/#creating-the-perfect-pr

### CODE REVIEW

**Review by bramkragten — DISMISSED** [review]

File: src/panels/config/automation/add-automation-element-dialog.ts [hunk 1]
```diff
@@ -376,20 +389,7 @@ class DialogAddAutomationElement
...
-    this._selectedGroup = undefined;
-    this._selectedTarget = undefined;
-    this._tab = this._newTriggersAndConditions ? "targets" : "groups";
-    this._filter = "";
-    this._manifests = undefined;
-    this._domains = undefined;
-    this._bottomSheetMode = false;
-    this._narrow = false;
-    this._targetItems = undefined;
-    this._loadItemsError = false;
```
Comment (bramkragten) [inline src/panels/config/automation/add-automation-element-dialog.ts:392]: For memory reasons, I still think we should clean up most of these on close. At least the non boolean ones.

**Review by bramkragten — APPROVED** [review]
